frontend/public/components/marketplace/kubernetes-marketplace-items.jsx

```diff
@@ -208,7 +208,7 @@ export class MarketplaceTileViewPage extends React.Component {
       <CatalogTileView.Category totalItems={items.length} viewAll={true}>
         {_.map(items, item => {
           const { obj, name, imgUrl, iconClass, provider, description } = item;
```

Comment:
What do you think of adding uid to the ```normalizePackageManifests``` function in the kubernetes-marketplace.jsx file? And retrieving it here with
```javascript
    const { name, imgUrl, iconClass, provider, description, uid } = item;
```
That way all the extraction from the packageManifest is together?